raphtory/src/python/graph/graph.rs

```diff
@@ -459,6 +460,83 @@ impl PyGraph {
         Ok(graph.graph)
     }
 
+    /// Load a graph from Parquet file.
+    ///
+    /// Args:
+    ///     edge_parquet_path (str): Parquet file or directory of Parquet files containing the edges.
+    ///     edge_src (str): The column name for the source node ids.
+    ///     edge_dst (str): The column name for the destination node ids.
+    ///     edge_time (str): The column name for the timestamps.
+    ///     edge_properties (list): The column names for the temporal properties (optional) Defaults to None.
+    ///     edge_const_properties (list): The column names for the constant properties (optional) Defaults to None.
+    ///     edge_shared_const_properties (dict): A dictionary of constant properties that will be added to every edge (optional) Defaults to None.
+    ///     edge_layer (str): The edge layer name (optional) Defaults to None.
+    ///     layer_in_df (bool): Whether the layer name should be used to look up the values in a column of the edge_df or if it should be used directly as the layer for all edges (optional) defaults to True.
```

Comment:
Should this not be `layer_in_file` or `layer_in_parquet`?

Reply:
Also there are several references to dataframes here we should possible remove - will post in the chat to get opinions